package.json

```diff
@@ -68,8 +68,7 @@
   },
   "lint-staged": {
     "*.{ts,json,md}": [
-      "prettier --write",
-      "git add"
```

Comment:
This can be removed safely? What does it change?

Reply:
The `lint-staged` package already takes care of the `git add` command and, as such, doesn't need to be defined separately.